std/parallelism.d

```diff
@@ -986,9 +995,9 @@ uint totalCPUsImpl() @nogc nothrow @trusted
     }
     else version (useSysctlbyname)
     {
-        version (OSX)
+        version (Darwin)
         {
-            auto nameStr = "machdep.cpu.core_count\0".ptr;
+            enum nameStr = "hw.physicalcpu";
```

Comment:
Wouldn't `hw.logicalcpu` (and, previously, `machdep.cpu.thread_count`) be more consistent with the other values? The goal of `totalCPUs` is to return a good number for how many threads to create, and the number of logical CPUs is likely a better fit than the number of physical CPUs.

Reply:
I didn't want to change the semantics.